tests/ha/qnetd.pm

```diff
@@ -92,15 +92,23 @@ sub run {
         qdevice_status('started');
     }
 
+    # Ensure promotable resource is in node 1
+    script_run 'crm resource migrate promotable-1 ' . choose_node(1) if is_node(1);
+
     # Perform Split Brain test
     barrier_wait("SPLIT_BRAIN_TEST_READY_$cluster_name");
 
     record_info('Split-brain info', 'Split brain test');
 
+    # Stop stonith to prevent fencing of node before our check
+    assert_script_run "crm resource stop stonith-sbd" if is_node(1);
```

Comment:
I just add a test where I was surprised to find `stonith-sbd` disable, but it's fine in that case. But it could be a good idea to finally add a `record_info` note to explain this also in the test result, WDYT?

Reply:
Makes sense. Perhaps we can change the comment on L103 for a `record_info`. Do you have the link to the test?

Reply:
It was this test/screen: https://openqa.suse.de/tests/4342977#step/qnetd/50. We can see later that stonith-sbd is restarted: https://openqa.suse.de/tests/4342977#step/qnetd/53.
I just had to think and asked Julien for this xD

> Makes sense. Perhaps we can change the comment on L103 for a `record_info`. Do you have the link to the test?

Yes good idea and the easiest one.

Reply:
Ok, I will push it soon!
I took a long time to remember it too...